src/components/tests/__snapshots__/omni-panel-status-labels.test.tsx.snap

```diff
@@ -0,0 +1,216 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Component renders correctly 1`] = `
+<OmniPanelStatusLabels
+  modalLabelClass="mock-style"
+  modeText="State"
+  name="Robot"
+>
+  <WithStyles(ForwardRef(Typography))
+    className="makeStyles-hideText-1"
+    variant="h6"
+  >
+    <ForwardRef(Typography)
+      className="makeStyles-hideText-1"
+      classes={
+        Object {
+          "alignCenter": "MuiTypography-alignCenter",
+          "alignJustify": "MuiTypography-alignJustify",
+          "alignLeft": "MuiTypography-alignLeft",
+          "alignRight": "MuiTypography-alignRight",
+          "body1": "MuiTypography-body1",
+          "body2": "MuiTypography-body2",
+          "button": "MuiTypography-button",
+          "caption": "MuiTypography-caption",
+          "colorError": "MuiTypography-colorError",
+          "colorInherit": "MuiTypography-colorInherit",
+          "colorPrimary": "MuiTypography-colorPrimary",
+          "colorSecondary": "MuiTypography-colorSecondary",
+          "colorTextPrimary": "MuiTypography-colorTextPrimary",
+          "colorTextSecondary": "MuiTypography-colorTextSecondary",
+          "displayBlock": "MuiTypography-displayBlock",
+          "displayInline": "MuiTypography-displayInline",
+          "gutterBottom": "MuiTypography-gutterBottom",
+          "h1": "MuiTypography-h1",
+          "h2": "MuiTypography-h2",
+          "h3": "MuiTypography-h3",
+          "h4": "MuiTypography-h4",
+          "h5": "MuiTypography-h5",
+          "h6": "MuiTypography-h6",
+          "noWrap": "MuiTypography-noWrap",
+          "overline": "MuiTypography-overline",
+          "paragraph": "MuiTypography-paragraph",
+          "root": "MuiTypography-root",
+          "srOnly": "MuiTypography-srOnly",
+          "subtitle1": "MuiTypography-subtitle1",
+          "subtitle2": "MuiTypography-subtitle2",
+        }
+      }
+      variant="h6"
```

Comment:
I got to say that this was weird for me, but actually happened the same when I executed this

```
test('test', () => {
  expect(toJson(mount(<Typography variant="h4"> test </Typography>))).toMatchSnapshot();
});
```

Reply:
Hmmmm .... I suppose you mean that when you run the following test with the `h4` variant, the new test results are not reflected in the `.snap` file?

For me, when I ran this test, the results are appended at the end of the file, after the previous snapshot test.

Reply:
Sorry this was just a comment, it was weird to me to see that the component to prints its whole API/ options

```
classes={
        Object {
          "alignCenter": "MuiTypography-alignCenter",
          "alignJustify": "MuiTypography-alignJustify",
          "alignLeft": "MuiTypography-alignLeft",
          "alignRight": "MuiTypography-alignRight",
          "body1": "MuiTypography-body1",
          "body2": "MuiTypography-body2",
          "button": "MuiTypography-button",
          "caption": "MuiTypography-caption",
          "colorError": "MuiTypography-colorError",
          "colorInherit": "MuiTypography-colorInherit",
          "colorPrimary": "MuiTypography-colorPrimary",
          "colorSecondary": "MuiTypography-colorSecondary",
          "colorTextPrimary": "MuiTypography-colorTextPrimary",
          "colorTextSecondary": "MuiTypography-colorTextSecondary",
          "displayBlock": "MuiTypography-displayBlock",
          "displayInline": "MuiTypography-displayInline",
          "gutterBottom": "MuiTypography-gutterBottom",
          "h1": "MuiTypography-h1",
          "h2": "MuiTypography-h2",
          "h3": "MuiTypography-h3",
          "h4": "MuiTypography-h4",
          "h5": "MuiTypography-h5",
          "h6": "MuiTypography-h6",
          "noWrap": "MuiTypography-noWrap",
          "overline": "MuiTypography-overline",
          "paragraph": "MuiTypography-paragraph",
          "root": "MuiTypography-root",
          "srOnly": "MuiTypography-srOnly",
          "subtitle1": "MuiTypography-subtitle1",
          "subtitle2": "MuiTypography-subtitle2",
        }
```